en-US/mozorg/internet-health/privacy-security.lang

```diff
@@ -182,10 +285,15 @@ Cohosting <a href="%(standford)s" rel="external">talks with Stanford’s Center
 Hungry for more?
 
 
+# This string is obsolete. Don't remove.
 ;Keep reading about Privacy and Security in our <a href="%(healthreport)s">Internet Health Report</a>.
 Keep reading about Privacy and Security in our <a href="%(healthreport)s">Internet Health Report</a>.
 
 
+;Read on about Digital Inclusion in our <a href="%(healthreport)s">Internet Health Report</a>.
```

Comment:
This string should have been behind a tag. Add the tag `internet_health_subpage_rebrand`, I'll fix the code.